code/drasil-docLang/lib/Drasil/Sections/Requirements.hs

```diff
@@ -100,6 +100,17 @@ nfReqIntro :: [Contents] -> Contents
 nfReqIntro [] = mkParagraph $ emptySectSentPlu [nonfunctionalRequirement]
 nfReqIntro _  = mkParagraph $ reqIntroStart +:+. nfrReqIntroBody
 
+-- | Common Non-Functional Requirement for Maintainability.
+-- Takes in a Reference Address ('String'), a percent value ('Integer'), 
+-- and a label ('String').
+mkMaintainableNFR :: String -> Integer -> String -> ConceptInstance
+mkMaintainableNFR refAddress percent lbl = cic refAddress (foldlSent [
+  S "If a likely change is made" `S.toThe` 
+  S "finished software, it will take", addPercent percent, EmptyS `S.ofThe`
```

Comment:
Is this `EmptyS` needed? I feel like it can be removed, but maybe not

Reply:
Yep, you're right!